crates/uniffi-bindgen/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "uniffi-bindgen"
+version = "0.1.0"
+edition = "2021"
+
+[[bin]]
+name = "uniffi-bindgen"
+
+[dependencies]
+uniffi = { workspace = true, features = ["cli"] }
```

Comment:
It was available with the CLI feature and as a build dependency, is the latter no longer necessary?

Reply:
Yeah the build dependency was a mistake -- its only needed if you're defining types in a UDL file, not via the macros.